### ISSUE TITLE
great_expectations not found after successful pip install

### ISSUE THREAD

**Issue by ejstembler** [fivetran/great_expectations#4990]
**Describe the bug**

I'm following the [Install Great Expectations and dependencies](https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_setup#install-great-expectations-and-dependencies) step in the [Tutorial, Step 1: Setup](https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_setup).  After `pip install great_expectations` finishes successfully, the next step `great_expectations --version` fails on both macOS and Windows 10:

**macOS**:

```console
zsh: command not found: great_expectations
```

**Windows**:

```console
'great_expectations' is not recognized as an internal or external command, operable program or batch file.
```

**To Reproduce**

1. `git clone https://github.com/superconductive/ge_tutorials`
2. `cd ge_tutorials`
3. `pip install great_expectations`
4. `great_expectations --version`

**Expected behavior**

The expected output should be something like:

```console
great_expectations, version 0.15.3
```

**Environment (please complete the following information):**

* **macOS**
  * Monteray 12.3.1 (21E258)
  * Python 3.9.10
  * great_expectations 0.15.3
* **Windows**
  * 10 Version 21H1 (OS Build 19043.1682)
  * Python 3.8.10
  * great_expectations 0.15.3

**Comment by talagluck** [comment]
Hi @ejstembler - I'm not able to reproduce this issue. When I run the command, the output is as expected. While it seems unlikely, is it possible you have issues with your PATH on both Windows and MacOS? When you use `pip` to install other packages with CLIs, do those work as expected?

**Comment by ejstembler** [comment]
@talagluck Those two failures were on my work MacBook Pro (with Parallels VM for Windows).

I tried installing on my personal Mac mini and Windows gaming machine... in both cases on my home machines, pip had some permission errors when trying to install one of the dependencies.  If I used `sudo` for Mac and `Run As Administrator` for Windows, pip finally finished installing without any errors and the version statement works.

So, I'll try again on my work MacBook Pro (and Windows VM) later to see it turns out to be the same issue...

Thanks!
